packages/block-library/src/query/edit/enhanced-pagination-modal.js

```diff
@@ -37,24 +36,14 @@ export default function EnhancedPaginationModal( {
 		setOpen( false );
 	};
 
-	let notice = __(
-		'If you still want to prevent full page reloads, remove that block, then disable "Reload full page" again in the Query Block settings.'
-	);
-	if ( hasBlocksFromPlugins ) {
-		notice =
-			__(
-				'Currently, avoiding full page reloads is not possible when non-interactive or non-client Navigation compatible blocks from plugins are present inside the Query block.'
-			) +
-			' ' +
-			notice;
-	} else if ( hasPostContentBlock ) {
-		notice =
-			__(
-				'Currently, avoiding full page reloads is not possible when a Content block is present inside the Query block.'
-			) +
-			' ' +
-			notice;
-	}
+	const notice =
+		__(
+			'If you still want to prevent full page reloads, remove that block, then disable "Reload full page" again in the Query Block settings.'
+		) +
+		' ' +
+		__(
+			'Currently, avoiding full page reloads is not possible when non-interactive or non-client Navigation compatible blocks from plugins are present inside the Query block.'
+		);
```

Comment:
I wasn't sure if this should be a unique string or if it have to be kept splitted for translations.